src/blocks/VideoBlock/VideoBlock.fragment.graphql

```diff
@@ -0,0 +1,30 @@
+fragment VideoBlock on VideoBlockRecord {
+  id
+  videoAsset {
+    video {
+      duration
+      framerate
+      mp4Url
+      muxAssetId
+      muxPlaybackId
+      streamingUrl
+      thumbnailUrl(format: jpg)
+    }
+    width
+    height
+    alt
+    title
+  }
+  title
```

Comment:
Fragment has a `title` and a `videoAsset.title`. Maybe we only need the `videoAsset.title`?

Reply:
Using `title || videoAsset.title`.